lib/lotus/validations/attribute.rb

```
@@ -208,6 +209,17 @@ def nested
         end
       end
 
+      # Validates user defined validations
+      #
+      # @api private
+      def user_defined
+        @validations.each do |name, validation|
+          break unless validation.respond_to? :call
```

I would expect this to blow up instead of silently ignore the problem.